app/code/Magento/Backend/view/adminhtml/layout/adminhtml_dashboard_index.xml

```diff
@@ -10,9 +10,10 @@
     <body>
         <referenceContainer name="page.main.actions">
             <block class="Magento\Backend\Block\Store\Switcher" name="store_switcher" as="store_switcher" template="Magento_Backend::store/switcher.phtml">
-                <action method="setUseConfirm">
-                    <argument name="params" xsi:type="string">0</argument>
-                </action>
+                <arguments>
+                    <argument name="use_confirm" xsi:type="string">1</argument>
+                    <argument name="is_using_iframe" xsi:type="string">0</argument>
+                </arguments>
```

Comment:
Feels like it won't work, as we don't have this argument in constructor

Reply:
you can get it via Block class no problem by getData('variable')

Reply:
https://devdocs.magento.com/guides/v2.4/frontend-dev-guide/layouts/xml-instructions.html